exovetter/modshift/modshift.py

```diff
@@ -117,11 +117,13 @@ def compute_modshift_metrics(time, flux, model, period_days, epoch_days,
 
     results["false_alarm_threshold"] = compute_false_alarm_threshold(
         period_days, duration_hrs)
-    results["Fred"] = np.nan
+
+    results["Fred"] = np.std(conv) / np.std(bflux)  # orig code had np.nan
 
     if show_plot:
-        plotmodshift.plot_modshift(phi_days, flux, model, conv, results)
-    return results
+        plotmodshift.plot_modshift(phi_days, period_days, flux, model,
```

Comment:
phi_days runs from zero to period_days, so there's no need to pass in period_days as a separate argument, I don't think.

Reply:
You are suggesting using np.max(phi_days) to get period_days.  I believe that is only true if the data completely samples the orbit.  I'm not sure I want to make that assumption. 
I also think it is a pretty inaccurate way to get period_days as it depends on the sampling of the time input the vetter.